vite.config.ts

```diff
@@ -1,16 +1,17 @@
-import { defineConfig } from "vite";
+import { defineConfig, loadEnv } from "vite";
 import react from "@vitejs/plugin-react-swc";
 import tsconfigPaths from "vite-tsconfig-paths";
-import dotenv from "dotenv";
-
-dotenv.config();
 
 // https://vitejs.dev/config/
-export default defineConfig({
-  plugins: [react(), tsconfigPaths()],
-  server: {
-    proxy: {
-      "/api": process.env.API_SERVER_URL,
+export default defineConfig(({ mode }) => {
+  const TmpEnv = loadEnv(mode, process.cwd());
+
+  return {
+    plugins: [react(), tsconfigPaths()],
+    server: {
+      proxy: {
+        "/api": TmpEnv.VITE_API_SERVER_URL,
```

Comment:
여기도 같은 이유로 수정이 가능합니다!
```suggestion
        "/api": env.VITE_API_SERVER_URL,
```

Reply:
Rgr